src/komponenter/alleredeOpprettetTiltak/innholdsvisning/RadInfo.tsx

```diff
@@ -9,9 +8,11 @@ interface Props {
 
 const RadInfo: FunctionComponent<Props> = ({ label, info, infoNotBold }) => {
     return (
-        <div className={'alleredeOpprettetAvtale__rad-element'}>
+        <div className={'rad-element'}>
```

Comment:
er det sånn at `alleredeOpprettetAvtale.less` som definerer `rad-element` er importert i en annen tsx-fil som er rendret, så den er tilgjengelig her også?

Reply:
Stiler importert i less-filer på den måten vi gjør nå er tilgjengelig for hele applikasjonen!

Reply:
Endret denne til å ha en egen less-fil, og importerte den som en modul